lab_2_keywords_cooccurrence/main.py

```diff
@@ -29,7 +111,24 @@ def extract_candidate_keyword_phrases(phrases: Sequence[str], stop_words: Sequen
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not check_list(phrases, str) or not check_list(stop_words, str):
+        return None
+    lowed_phrases = []  # put lower phrases
+    for idx, phrs in enumerate(phrases):
+        lowed_phrases.append(phrs.lower())
```

Comment:
do `split` and `lower` before adding to the list

Reply:
Done